app/views/projects/blob/_actions.html.haml

```diff
@@ -17,6 +17,10 @@
         tree_join(@commit.sha, @path)), class: 'btn btn-sm'
 
 - if allowed_tree_edit?
-  = button_tag class: 'remove-blob btn btn-sm btn-remove',
+  = button_tag class: 'btn btn-small btn-primary btn-xs',
+      'data-toggle' => 'modal', 'data-target' => '#modal-replace-blob' do
+    Replace	 
+  = |
```

Comment:
@stanhu @tsigo , It can be done by button group, and I attach one screen shot, and will update the code to the repository, thanks.
![chinesereminder1](https://cloud.githubusercontent.com/assets/7370334/8374439/a0c99e18-1bf5-11e5-89bb-41d8797b7737.png)